scheds/rust/scx_layered/src/main.rs

```diff
@@ -1170,11 +1174,11 @@ impl<'a> Scheduler<'a> {
                         }
                         LayerMatch::NiceAbove(nice) => {
                             mt.kind = bpf_intf::layer_match_kind_MATCH_NICE_ABOVE as i32;
-                            mt.nice_above_or_below = *nice;
+                            mt.nice = *nice;
                         }
                         LayerMatch::NiceBelow(nice) => {
                             mt.kind = bpf_intf::layer_match_kind_MATCH_NICE_BELOW as i32;
-                            mt.nice_above_or_below = *nice;
+                            mt.nice = *nice;
```

Comment:
Huh, will this `match` compile without having a `LayerMatch::NiceEquals` case?

Reply:
fixed